dev_tools/conf/pip-list-dev-tools.txt

```diff
@@ -1,28 +1,27 @@
 # For testing and analyzing code.
-mypy~=0.701.0
-pylint~=2.3.0
-pytest~=3.8.0
-pytest-asyncio~=0.10.0
-pytest-cov~=2.5.0
-pytest-benchmark~=3.2.0
-yapf~=0.27.0
+mypy~=0.750
```

Comment:
This change is wrong. It means that when mypy introduces new checks our build will break.

Reply:
do you want a == and exact version pin?